store/search/src/main/scala/org/apache/spark/rpc/Master.scala

```diff
@@ -142,7 +142,7 @@ class Master(sparkConf: SparkConf) {
       (address, schedulable.ref.ask[ShutdownResponse](ShutdownRequest("user")))
     }
     futures.foreach { case (address, future) =>
-      ThreadUtils.awaitResult(future, Duration.apply("10s"))
+      ThreadUtils.awaitResult(future, Duration.apply("100s"))
```

Comment:
Please add a CarbonProperty for search mode, and make the default as 10

Reply:
ok, down, default value is 10s